docs/Developers/External_Bears.rst

```diff
@@ -55,7 +55,7 @@ Writing a Bear in C++
 ---------------------
 
 The bear that will be created with this tutorial will check whether there is
-any **coala** spelled with a capital ``C`` since that is a horrible mistake for
+any **coala** spelt with a capital ``C`` since that is a horrible mistake for
```

Comment:
again, this is the British English way of spelling `spelled` - don't think it is a necessary change.